src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.Unix.cs

```diff
@@ -70,6 +70,7 @@ internal bool BackendSupportsCustomCertificateHandling
         }
 
         [Theory]
+        [ActiveIssue(28002)]
```

Comment:
Should this be disabled everywhere like this?  I'm wondering if it should actually just be a condition in the test, e.g.
```C#
if (expectedFailure && UseSocketsHttpHandler && RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
{
    return; // [ActiveIssue(28002)]
}
```
or something along those lines.

Reply:
I assume we want to do this is because we don't want to lose the coverage for the succeed test data?

Reply:
> I assume we want to do this is because we don't want to lose the coverage for the succeed test data?

Right, we would ideally only disable tests for the narrowest circumstances in which they're failing.